tools/SourceKit/lib/SwiftLang/SwiftLangSupport.cpp

```diff
@@ -545,21 +545,24 @@ UIdent SwiftLangSupport::getUIDForSymbol(SymbolInfo sym, bool isRef) {
     return UID_FOR(TypeAlias);
 
   case SymbolKind::Function:
+  case SymbolKind::StaticMethod:
     if (sym.SubKind == SymbolSubKind::SwiftPrefixOperator)
       return UID_FOR(FunctionPrefixOperator);
     if (sym.SubKind == SymbolSubKind::SwiftPostfixOperator)
       return UID_FOR(FunctionPostfixOperator);
     if (sym.SubKind == SymbolSubKind::SwiftInfixOperator)
       return UID_FOR(FunctionInfixOperator);
-    return UID_FOR(FunctionFree);
+    if (sym.Kind == SymbolKind::StaticMethod) {
```

Comment:
This feels a bit wonky to me - I had originally created different UIDs to prevent putting the cases together but ended up thinking it was unnecessary. I appreciate better ideas! 🙂